vcr/signature/ldutils.go

```diff
@@ -42,6 +57,21 @@ func NewEmbeddedFSDocumentLoader(fs embed.FS, nextLoader ld.DocumentLoader) ld.D
 	}
 }
 
+// NewFilteredLoader accepts a list of allowed urls and a nextLoader and creates a new filteredDocumentLoader
```

Comment:
Comment states the obvious which is already stated by the function signature, better explain the intention/working of the returned type:

```suggestion
// NewFilteredLoader creates a (wrapper) DocumentLoader that only loads from the given accepts list of allowed URLs
```